src/node_worker.cc

```diff
@@ -132,7 +132,16 @@ class WorkerThreadData {
  public:
   explicit WorkerThreadData(Worker* w)
     : w_(w) {
-    CHECK_EQ(uv_loop_init(&loop_), 0);
+    int ret = uv_loop_init(&loop_);
+    if (ret != 0) {
+      char err_buf[128];
+      uv_err_name_r(ret, err_buf, sizeof(err_buf));
+      std::string error_str = SPrintF("%s %s",
+        "ERR_WORKER_INIT_FAILED:",
+        err_buf);
```

Comment:
Can you line up the arguments? Also, the first %s doesn't really make sense when the corresponding argument is a static string. Just write `SPrintF("ERR_WORKER_INIT_FAILED: %s", err_buf)`.

Reply:
@bnoordhuis, thanks. Addressed your suggestions.